tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -172,11 +239,59 @@ def test_45519(self):
         LOGGER.info("STARTED: Test Delete source object when copy simple object is in progress in "
                     "the same bucket")
         self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.srcbuck,
-                                                         "dest-obj", "NoSuchKey"),
+                                                         "dest-obj", errmsg.NO_SUCH_KEY_ERR),
                                                         (self.srcbuck, "src-obj"))
         LOGGER.info("ENDED: Test Delete source object when copy simple object is in progress in "
                     "the same bucket")
 
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45520")
+    @CTFailOn(error_handler)
```

Comment:
This decorator has not effects so you can remove it wherever used in this PR.

Reply:
done . Removed the decorator from all places .